tests/ext/drv/test_ext.c

```diff
@@ -0,0 +1,489 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+/*++
+
+Abstract:
+    Implementation of hook and program information NPI providers registration and unregistration.
+
+Environment:
+
+    Kernel mode
+
+--*/
+#define INITGUID
+
+#include <guiddef.h>
+#include <ntddk.h>
+
+#include "ebpf_platform.h"
+#include "ebpf_program_types.h"
+#include "ebpf_windows.h"
+
+#include "test_ext_helpers.h"
+
+#define RTL_COUNT_OF(arr) (sizeof(arr) / sizeof(arr[0]))
```

Comment:
We already have EBPF_COUNT_OF() in ebpf_platform.h.  Just use that.

Reply:
This macro was not needed. I removed it.